touchevents.html

```diff
@@ -130,70 +130,43 @@
+      </p>
     </section>
 
     <section id='abstract'>
-      The Touch Events specification defines a set of low-level events that
-      represent one or more points of contact with a touch-sensitive surface,
-      and changes of those points with respect to the surface and any DOM
-      elements displayed upon it (e.g. for touch screens) or associated with it
-      (e.g. for drawing tablets without displays).  It also addresses
-      pen-tablet devices, such as drawing tablets, with consideration toward
-      stylus capabilities.
+      <p>
+        The Touch Events specification defines a set of low-level events that represent one or more points of contact with a touch-sensitive surface, and changes of those points with respect to the surface and any DOM elements displayed upon it (e.g. for touch screens) or associated with it (e.g. for drawing tablets without displays). It also addresses pen-tablet devices, such as drawing tablets, with consideration toward stylus capabilities.
+      </p>
     </section>
 
     <section id='introduction' class='informative'>
       <h2>Introduction</h2>
 
       <p>
-        User Agents that run on terminals which provide touch input to use web
-        applications typically use interpreted mouse events to allow users
-        to access interactive web applications. However, these interpreted
-        events, being normalized data based on the physical touch input, tend
-        to have limitations on delivering the intended user experience.
-        Additionally, it is not possible to handle concurrent input regardless
-        of device capability, due to constraints of mouse events: both
-        system level limitations and legacy compatibility.
+        User Agents that run on terminals which provide touch input to use web applications typically use interpreted mouse events to allow users to access interactive web applications. However, these interpreted events, being normalized data based on the physical touch input, tend to have limitations on delivering the intended user experience. Additionally, it is not possible to handle concurrent input regardless of device capability, due to constraints of mouse events: both system level limitations and legacy compatibility.
       </p>
 
       <p>
-        Meanwhile, native applications are capable of handling both cases with
-        the provided system APIs.
+        Meanwhile, native applications are capable of handling both cases with the provided system APIs.
       </p>
 
       <p>
-        The Touch Events specification provides a solution to this problem by
-        specifying interfaces to allow web applications to directly handle touch
-        events, and multiple touch points for capable devices.
+        The Touch Events specification provides a solution to this problem by specifying interfaces to allow web applications to directly handle touch events, and multiple touch points for capable devices.
       </p>
 
+      <!--
```

Comment:
Fine with me to just remove this entirely rather than leave comment cruft in the code.  This is what revision tracking is for :-)